converter.py

```diff
@@ -397,14 +397,32 @@ def generate_question(row, columns, question_translations):
         "concept": question_concept_id
     }
 
+    # Store min/max values if rendering is numeric/number
+    min_value = None
+    max_value = None
+    if question_rendering in ['numeric', 'number']:
+        if 'Lower limit' in columns and pd.notnull(row['Lower limit']):
+            min_value = int(row['Lower limit'])
+            question_options['min'] = min_value
+        if 'Upper limit' in columns and pd.notnull(row['Upper limit']):
+            max_value = int(row['Upper limit'])
+            question_options['max'] = max_value
+
     if should_render_workspace(question_rendering):
         workspace_button_label = get_workspace_button_label(question_rendering)
         question.pop('type')
-        question_options = {
+        # Create new question_options while preserving min/max
+        updated_options = {
             "rendering": "workspace-launcher",
             "buttonLabel": workspace_button_label,
             "workspaceName": question_rendering
         }
+        # Add back min/max if they existed
+        if min_value is not None:
+            updated_options['min'] = min_value
+        if max_value is not None:
+            updated_options['max'] = max_value
+        question_options = updated_options
```

Comment:
I don't think these changes are necessary for workspaces. At the moment, data for workspaces aren't handled within the form engine so there's no way of validating it based on these params

Reply:
Good catch, I had to modify should_render_workspace() to include 'numeric' in the list of non-workspace rendering options which was missing. So this should prevent numeric fields from being incorrectly converted to workspace launchers and allows min/max values to be properly included in the form schema.